src/dlgConnectionProfiles.cpp

```diff
@@ -1778,140 +1765,128 @@ void dlgConnectionProfiles::slot_connectToServer()
 
 bool dlgConnectionProfiles::validateProfile()
 {
-    bool valid = true;
-
-    notificationArea->hide();
-    notificationAreaIconLabelWarning->hide();
-    notificationAreaIconLabelError->hide();
-    notificationAreaIconLabelInformation->hide();
-    notificationAreaMessageBox->setText(tr(""));
+    validProfile = true;
 
     QListWidgetItem* pItem = profiles_tree_widget->currentItem();
 
     if (pItem) {
         QString name = profile_name_entry->text().trimmed();
+        profile_name_entry->setStyleSheet("");
         const QString allowedChars = QStringLiteral(". _0123456789-#&aAbBcCdDeEfFgGhHiIjJkKlLmMnNoOpPqQrRsStTuUvVwWxXyYzZ");
-
+        removeNotification(tr("Please only use one of the following characters:\"%1\".").arg(allowedChars));
```

Comment:
Would `Please, only use the following characters:`... be better - as they are like to want more than a *single* character overall... :open_mouth: